exercises/practice/list-ops/.meta/src/example.cr

```diff
@@ -0,0 +1,59 @@
+class ListOps(T)
```

Comment:
I think this would be better as a `module` without instance methods. It's strange to initialize an object without state in order to be able to use these functions.